setup.py

```diff
@@ -35,7 +35,7 @@
         "Topic :: Terminals"],
     packages=find_packages() + ['axiom.plugins', 'xmantissa.plugins'],
     install_requires=[
-        "twisted>=14.0.0",
+        "twisted>=14.0.0,<16.4.0",
```

Comment:
Yep. I put a comment on there. Instead of adding a version constraint, I'm just adding a workaround and linking to the issue.